src/Model/BunqModel.php

```diff
@@ -270,7 +251,7 @@ private static function determineJsonResponseValue(array $responseArray)
     protected static function processForUuid(BunqResponseRaw $responseRaw)
```

Comment:
Return type? 😄

Reply:
This is [WIP]; return types are coming :P